test/e2e/instrumentation-hook/instrumentation-hook.test.ts

```diff
@@ -0,0 +1,61 @@
+import { createNextDescribe } from 'e2e-utils'
+import { check } from 'next-test-utils'
+
+createNextDescribe(
+  'instrumentation-hook',
+  {
+    files: __dirname,
+    nextConfig: {
+      experimental: {
+        instrumentationHook: true,
+      },
+    },
```

Comment:
Either want `skipDeployment: true` or manually skipping `cliOutput` checks for `global.isNextDeploy`